github/orgs_teams.go

```diff
@@ -226,14 +214,15 @@ func (s *OrganizationsService) ListTeamRepos(team int, opt *ListOptions) ([]Repo
 // permissions team has for that repo.
 //
 // GitHub API docs: http://developer.github.com/v3/orgs/teams/#get-team-repo
```

Comment:
Thanks, @shurcooL!  TL;DR: I'll delete the first link.
One possible option is to replace it with https://developer.github.com/v3/orgs/teams/#list-team-repos since it describes this call without the custom media type header, but when comparing the docs content of the two links, it seems to me that the first link isn't very useful compared to the second.  I'll delete it.